include/theme.h

```diff
@@ -30,6 +31,14 @@ struct theme_snapping_overlay {
 	float border_color[3][4];
 };
 
+enum lab_button_state {
+	LAB_BS_HOVER = 1 << 0,
+	LAB_BS_TOGGLED = 1 << 1,
+	LAB_BS_ROUNDED = 1 << 2,
+};
+
+#define LAB_NR_BUTTON_STATE_SETS (1 << 3)
```

Comment:
I replaced it with an enum entry `LAB_BS_ALL = LAB_BS_HOVER | LAB_BS_TOGGLED | LAB_BS_ROUNDED,` and changed the array definitions to `buttons[LAB_SSD_BUTTON_LAST + 1][LAB_BS_ALL + 1];` since I feel having `LAB_NR_BUTTON_STATE_SETS` as an enum entry alongside `LAB_BS_{HOVER,TOGGLED,ROUNDED}` a bit off. Other suggestions are welcome.